README.md

```diff
@@ -71,9 +71,25 @@ Babel does the transpiling from target to es5 browser compatible code, tsc does
 
 <a name="run"></a>
 
-## 4. Running
+## 4. Developing
```

Comment:
This is good. I was thinking the following part as mentioned [here](https://github.com/jayeclark/mycovidconnect#3-installation) doesn't make sense. :
```
docker-compose -f "docker-compose.yml" up -d --build
```

If this command doesn't work, run

`PORT=9119 docker-compose up`

this is more about how to build & run it on your local. the installation section is a pre-requisite to have the necessary stack before you can run the above :point_up:

Reply:
The docker app is a bit difficult to work with as I'm not able to easily edit the .env values, and the created .env file from the docker command doesn't 'live' in my local directory... and then to edit any .env variables, I need to rebuild the entire app (which takes several minutes), vs just quickly relaunching the dev server. I guess I also just prefer building things directly so I have more control over troubleshooting.

I'm planning to test some changes to dockerfile and docker-compose.yml a little as well, but the items in this PR were the few things that stood out when trying to build it locally the 'traditional' way (knowing that I already have the stack in place for most react projects.)

Reply:
it doesn't exist. Dockerfile assumes you created it. See: https://github.com/Xcov19/mycovidconnect/blob/36962c3a2e745c46ff4da0f06a02b9d8947c38eb/Dockerfile#L38

What you could do is, add this info in documentation and ask contributors to refer to .env.example